h2o-core/src/main/java/water/persist/PersistManager.java

```diff
@@ -465,6 +476,8 @@ public boolean mkdirs(String path) {
       validateHdfsConfigured();
       boolean b = I[Value.HDFS].mkdirs(path);
       return b;
+    } else if (isGcsPath(path)){
+      return true; // GCS doesn't support directories, so we don't need them to exist
```

Comment:
This behavior should be tested. There are some notes about [subdirectories in GCS](https://cloud.google.com/storage/docs/gsutil/addlhelp/HowSubdirectoriesWork) to be found. Are we sure we do not support those ?

What if the dir is not created and we return true ? Shouldn't we return false, or even UnsupportedOperationException ?

Reply:
The entire directory concept doesn't exist in GCS. 

> To the service, the object gs://your-bucket/abc/def.txt is just an object that happens to have "/" characters in its name.

My reasoning behind this `return true;` here is that we don't need to have directory created the way we do need in other file systems and throwing exception / returning false requires us to treat GCS exceptionally in every code which uses `PersistManager`

Reply:
Rethought it!
We should have something for bucket creation in this place, since the path might point at the bucket. 
Will take care later.

Reply:
Implemented `mkdirs`, so that bucket is created if doesn't exist. Directory within the bucket is ignored, since this concept is not supported by GCS.